<html>
<head>
  <title>Pathogen Informatics Repos</title>
  <link rel="stylesheet" href="assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="https://cdn.datatables.net/1.10.10/css/dataTables.bootstrap.min.css">
  <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
  <script src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js"></script>
  <script src="https://cdn.datatables.net/1.10.10/js/dataTables.bootstrap.min.js"></script>
</head>
<body>
  <div class="container">
    <div class="row page-header">
      <div class="col-md-4">
	  <img alt="Wellcome Trust Sanger Institute Logo" width=300 src="assets/img/logo.png">
	</div>
	<div class="col-md-8">
	  <h2 class="media-heading">Wellcome Trust Sanger Institute<br>Pathogen Informatics</h2>
	  <p>A list of our favorite repos from <a href="https://www.github.com/sanger-pathogens">Pathogen Informatics</a></p>
	</div>
    </div>
    <div class="row">
      <div class="col-md-12">
        <h2>Featured Repos</h2>
      </div>
    </div>
    <div id="featured_repos" class="row">
      
      <div class="col-md-4">
        <div class="panel panel-default">
          <div class="panel-heading"><a href="http://www.sanger.ac.uk/resources/software/artemis/"><h4>Artemis</h4></a></div>
	  <div class="panel-body">Artemis is a free genome viewer and annotation tool that allows visualization of sequence features and the results of analyses within the context of the sequence, and its six-frame translation</div>
	  <div class="panel-footer">
	    Last Updated: <span class='date'>2016-04-20T10:16:06Z</span>
	    
          </div>
        </div>
      </div>
      
      <div class="col-md-4">
        <div class="panel panel-default">
          <div class="panel-heading"><a href="http://sanger-pathogens.github.io/Roary"><h4>Roary</h4></a></div>
	  <div class="panel-body">Rapid large-scale prokaryote pan genome analysis</div>
	  <div class="panel-footer">
	    Last Updated: <span class='date'>2016-04-19T09:24:26Z</span>
	    
	    <br>Latest Release: v3.6.1
	    
          </div>
        </div>
      </div>
      
      <div class="col-md-4">
        <div class="panel panel-default">
          <div class="panel-heading"><a href="http://sanger-pathogens.github.io/circlator/"><h4>circlator</h4></a></div>
	  <div class="panel-body">A tool to circularize genome assemblies</div>
	  <div class="panel-footer">
	    Last Updated: <span class='date'>2016-04-19T12:13:40Z</span>
	    
	    <br>Latest Release: v1.2.1
	    
          </div>
        </div>
      </div>
      
    </div>
    <div class="row">
      <div class="col-md-12">
        <h2>Other Repos</h2>
      </div>
    </div>
    <div class="row">
      <div class="col-md-12">
        <table id="other_repos" class="table">
          <thead>
            <th>Repository</th>
            <th>Description</th>
	    <th>Score</th>
          </thead>
          <tbody>
          
            <tr>
              <td><a href="http://sanger-pathogens.github.io/iva/">iva</a></td>
              <td>de novo virus assembler of Illumina paired reads</td>
	      <td>57.579</td>
            </tr>
          
            <tr>
              <td><a href="http://sanger-pathogens.github.io/gubbins/">gubbins</a></td>
              <td>Rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences using Gubbins</td>
	      <td>37.948</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/ariba">ariba</a></td>
              <td>Antibiotic Resistance Identification By Assembly</td>
	      <td>35.114</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/rapid_annotation_transfer_tool">rapid_annotation_transfer_tool</a></td>
              <td>RATT: A tool for transferring genomic annotations between genomes</td>
	      <td>10.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Fastaq">Fastaq</a></td>
              <td>Python3 scripts to manipulate FASTA and FASTQ files</td>
	      <td>4.595</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Tradis">Bio-Tradis</a></td>
              <td>A set of tools to analyse the output from TraDIS analyses</td>
	      <td>4.373</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/pymummer">pymummer</a></td>
              <td>Python3 module for running MUMmer and reading the output</td>
	      <td>4.092</td>
            </tr>
          
            <tr>
              <td><a href="http://biorxiv.org/content/early/2016/01/29/038190">snp-sites</a></td>
              <td>Finds SNP sites from a multi fasta alignment file</td>
	      <td>4.033</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/MIDAS">MIDAS</a></td>
              <td>Front end for the HICF pathogen genome store</td>
	      <td>3.903</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/panito">panito</a></td>
              <td>Calculate genome wide average nucleotide identity (gwANI) for a multiFASTA alignment</td>
	      <td>3.75</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/bio_assembly_refinement">bio_assembly_refinement</a></td>
              <td>Tools to refine/finish an assembly</td>
	      <td>3.739</td>
            </tr>
          
            <tr>
              <td><a href="http://www.genedb.org/">GeneDB</a></td>
              <td>Web front-end for genomic data, developed at the Pathogen Genomics group, Wellcome Trust Sanger Institute</td>
	      <td>3.71</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/gff3toembl">gff3toembl</a></td>
              <td>Converts PROKKA GFF3 files to EMBL files for uploading annotated assemblies to EBI</td>
	      <td>3.355</td>
            </tr>
          
            <tr>
              <td><a href="http://companion.sanger.ac.uk">companion</a></td>
              <td>Pipeline for automatic eukaryotic parasite annotation</td>
	      <td>3.2</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/assembly_improvement">assembly_improvement</a></td>
              <td>Improve the quality of a denovo assembly by scaffolding and gap filling</td>
	      <td>2.959</td>
            </tr>
          
            <tr>
              <td><a href="http://sanger-pathogens.github.io/pathogens-vm/">pathogens-vm</a></td>
              <td>To build a virtual machine with some Sanger pathogens software installed</td>
	      <td>2.56</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/BioPericles">BioPericles</a></td>
              <td>A work in progress. BioPericles is designed to use supervised Machine Learning techniques to classify samples into clusters</td>
	      <td>2.208</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/sanger-pathogens.github.io">sanger-pathogens.github.io</a></td>
              <td>Summary of Sanger Pathogen's Repos</td>
	      <td>1.975</td>
            </tr>
          
            <tr>
              <td><a href="http://www.genedb.org/web-artemis/">Web-Artemis</a></td>
              <td></td>
	      <td>1.902</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/update_pipeline">update_pipeline</a></td>
              <td></td>
	      <td>1.874</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/SnpEffWrapper">SnpEffWrapper</a></td>
              <td>Takes a VCF and applies annotations from a GFF using SnpEff</td>
	      <td>1.728</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/assembly-stats">assembly-stats</a></td>
              <td>Get assembly statistics from FASTA and FASTQ files</td>
	      <td>1.706</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Track-Schema">Bio-Track-Schema</a></td>
              <td>DBIC schema for pathogens tracking databases</td>
	      <td>1.67</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/json2email">json2email</a></td>
              <td>Takes a jinja2 template and some json and sends an email</td>
	      <td>1.578</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/citation_reporter">citation_reporter</a></td>
              <td>Website to show your team's publications (which are in PubMed)</td>
	      <td>1.533</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/primer3tools">primer3tools</a></td>
              <td>Design PCR primers and check uniqueness across multiple genomes</td>
	      <td>1.378</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Assembly_tools">Assembly_tools</a></td>
              <td>Scripts relating to genome assemblies</td>
	      <td>1.37</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/annotation_store">annotation_store</a></td>
              <td></td>
	      <td>1.37</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Metadata-Validator">Bio-Metadata-Validator</a></td>
              <td>Validate a genome metadata manifest against a checklist</td>
	      <td>1.229</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/eupathtables">eupathtables</a></td>
              <td>Python interface for reading and converting EuPathDB flat file dumps</td>
	      <td>1.203</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/DataPages">DataPages</a></td>
              <td>Static pages with convenient links to WTSI's pathogen sequencing data</td>
	      <td>1.191</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/VarGeneAnalysis">VarGeneAnalysis</a></td>
              <td>Code associated with some VarGene Analysis</td>
	      <td>1.173</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/companion-publication">companion-publication</a></td>
              <td>Accompanying information for the Companion publication</td>
	      <td>1.168</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Sequencescape-Schema">Bio-Sequencescape-Schema</a></td>
              <td>DBIC schema for the sequencescape warehouse database</td>
	      <td>1.097</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/ABACAS2">ABACAS2</a></td>
              <td>ABACAS2, a tool for ordering and orientating biosequences along a reference</td>
	      <td>1.003</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/PySnpSites">PySnpSites</a></td>
              <td>Experiments optimising python (with Cython)</td>
	      <td>1.002</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/structure">structure</a></td>
              <td>Copy of Structure 2.3.4 with patch applied to fix seg fault</td>
	      <td>1.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-HICF-Schema">Bio-HICF-Schema</a></td>
              <td></td>
	      <td>0.989</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-RetrieveAssemblies">Bio-RetrieveAssemblies</a></td>
              <td></td>
	      <td>0.687</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/mlst_check">mlst_check</a></td>
              <td></td>
	      <td>0.671</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/vr-codebase">vr-codebase</a></td>
              <td>The overall codebase developed and used by the Vertebrate Resequencing group at the Sanger Institute</td>
	      <td>0.37</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Farmpy">Farmpy</a></td>
              <td> Python3 package to handle job submission to a compute farm</td>
	      <td>0.332</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/trait_assembler">trait_assembler</a></td>
              <td></td>
	      <td>0.289</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/crawl2">crawl2</a></td>
              <td>A spiritual successor to Crawl</td>
	      <td>0.28</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Path-Find">Bio-Path-Find</a></td>
              <td></td>
	      <td>0.252</td>
            </tr>
          
            <tr>
              <td><a href="www.genedb.org">crawl</a></td>
              <td>Chado Restful Access Webservice Layer</td>
	      <td>0.25</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/deployment">deployment</a></td>
              <td></td>
	      <td>0.249</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Metagenomics">Bio-Metagenomics</a></td>
              <td></td>
	      <td>0.237</td>
            </tr>
          
            <tr>
              <td><a href="http://companion.sanger.ac.uk">annot-web</a></td>
              <td>Web interface for Companion</td>
	      <td>0.224</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-RNASeq">Bio-RNASeq</a></td>
              <td>The new Sanger Pathogen Informatics RNA Seq analysis pipeline</td>
	      <td>0.221</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/QUASR">QUASR</a></td>
              <td></td>
	      <td>0.208</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/resistome-db">resistome-db</a></td>
              <td></td>
	      <td>0.208</td>
            </tr>
          
            <tr>
              <td><a href="http://ccb.jhu.edu/software/kraken/">kraken</a></td>
              <td>Kraken taxonomic sequence classification system</td>
	      <td>0.2</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/fastml">fastml</a></td>
              <td>Addtional functionality for fastml, see http://fastml.tau.ac.il</td>
	      <td>0.2</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/vr-pipe">vr-pipe</a></td>
              <td>Generic pipeline system</td>
	      <td>0.2</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-AutomatedAnnotation">Bio-AutomatedAnnotation</a></td>
              <td>Perl module to take in an genomic assembly and produce annoation</td>
	      <td>0.149</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/pipelines_reporting">pipelines_reporting</a></td>
              <td></td>
	      <td>0.138</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/reference_track">reference_track</a></td>
              <td></td>
	      <td>0.137</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/setup_tracking">setup_tracking</a></td>
              <td>setup a vrtracking pipeline</td>
	      <td>0.137</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/vrtrack_crawl">vrtrack_crawl</a></td>
              <td></td>
	      <td>0.137</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/find_files_for_deletion">find_files_for_deletion</a></td>
              <td></td>
	      <td>0.137</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/gap5_consistency_checker">gap5_consistency_checker</a></td>
              <td></td>
	      <td>0.137</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/PathPipeConfTools">PathPipeConfTools</a></td>
              <td>Outputs details of all of the pathogen jobs in the pipeline</td>
	      <td>0.121</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Farm_blast">Farm_blast</a></td>
              <td>Python3 module to run blast+ or blastall in parallel on an LSF compute farm</td>
	      <td>0.121</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/validate_tools">validate_tools</a></td>
              <td>File validation scripts and tools</td>
	      <td>0.121</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/PathFind">PathFind</a></td>
              <td>Modularised versions of *find scripts</td>
	      <td>0.107</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/iva-publication">iva-publication</a></td>
              <td>Supplementary scripts and data for the IVA publication</td>
	      <td>0.1</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/fasta_grep">fasta_grep</a></td>
              <td>Efficiently pull filter FASTA files</td>
	      <td>0.1</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-ENA-DataSubmission">Bio-ENA-DataSubmission</a></td>
              <td></td>
	      <td>0.087</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-VertRes-Config">Bio-VertRes-Config</a></td>
              <td></td>
	      <td>0.086</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/pathogen-informatics-training">pathogen-informatics-training</a></td>
              <td></td>
	      <td>0.049</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/bragv">bragv</a></td>
              <td></td>
	      <td>0.037</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/HICF_checklist">HICF_checklist</a></td>
              <td></td>
	      <td>0.024</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-InterProScanWrapper">Bio-InterProScanWrapper</a></td>
              <td></td>
	      <td>0.021</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/concordance">concordance</a></td>
              <td></td>
	      <td>0.002</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-VertRes-Permissions">Bio-VertRes-Permissions</a></td>
              <td></td>
	      <td>0.002</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/LSF_jobrunner_Roary">LSF_jobrunner_Roary</a></td>
              <td></td>
	      <td>0.001</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-TypingGenerator">Bio-TypingGenerator</a></td>
              <td></td>
	      <td>0.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/remove_blocks_from_aln">remove_blocks_from_aln</a></td>
              <td></td>
	      <td>0.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Pequeno">Bio-Pequeno</a></td>
              <td></td>
	      <td>0.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-HPS-FastTrack">Bio-HPS-FastTrack</a></td>
              <td></td>
	      <td>0.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Resistome">Bio-Resistome</a></td>
              <td></td>
	      <td>0.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/annotations_update">annotations_update</a></td>
              <td></td>
	      <td>0.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-GFFValidator">Bio-GFFValidator</a></td>
              <td></td>
	      <td>0.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Pipeline-Comparison">Bio-Pipeline-Comparison</a></td>
              <td></td>
	      <td>0.0</td>
            </tr>
          
          </tbody>
        </table>
      </div>
    </div>
    <div class="row">
      <div class="col-md-12">
        <p>Data collected: <span class='date'>2016-04-21T19:07:46Z</span></p>
      </div>
    </div>
    <script>
      $(document).ready(function() {
	function timeDeltaString (millis) {
          var seconds = millis / 1000;
	  if (seconds < 120) {
            return Math.round(seconds) + ' seconds ago';
	  }
	  var minutes = seconds / 60;
	  if (minutes < 120) {
            return Math.round(minutes) + ' minutes ago';
	  }
	  var hours = minutes / 60;
	  if (hours < 48) {
            return Math.round(hours) + ' hours ago';
	  }
	  var days = hours / 24;
	  if (days < 28) {
            return Math.round(days) + ' days ago';
	  }
	  var weeks = days / 7;
	  if (weeks < 12) {
            return Math.round(weeks) + ' weeks ago';
	  }
	  var months = weeks / 4;
	  return Math.round(months) + ' months ago';
        }
        var now = new Date();
        $( "span.date" ).each(function() {
          var date_string = this.textContent;
	  var date_obj = new Date(date_string);
          this.textContent = timeDeltaString(now - date_obj);
	});
	$('#other_repos').DataTable({
	  "order": [[ 2, "desc" ]]
	});
      });
    </script>
  </div>
</body>
