<html>
<head>
  <title>Pathogen Informatics Repos</title>
  <link rel="stylesheet" href="assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="https://cdn.datatables.net/1.10.10/css/dataTables.bootstrap.min.css">
  <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
  <script src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js"></script>
  <script src="https://cdn.datatables.net/1.10.10/js/dataTables.bootstrap.min.js"></script>
</head>
<body>
  <div class="container">
    <div class="row page-header">
      <div class="col-md-4">
	  <img alt="Wellcome Trust Sanger Institute Logo" width=300 src="assets/img/logo.png">
	</div>
	<div class="col-md-8">
	  <h2 class="media-heading">Wellcome Trust Sanger Institute<br>Pathogen Informatics</h2>
	  <p>A list of our favorite repos from <a href="https://www.github.com/sanger-pathogens">Pathogen Informatics</a></p>
	</div>
    </div>
    <div class="row">
      <div class="col-md-12">
        <h2>Featured Repos</h2>
      </div>
    </div>
    <div id="featured_repos" class="row">
      
      <div class="col-md-4">
        <div class="panel panel-default">
          <div class="panel-heading"><a href="http://sanger-pathogens.github.io/Roary"><h4>Roary</h4></a></div>
	  <div class="panel-body">Rapid large-scale prokaryote pan genome analysis</div>
	  <div class="panel-footer">
	    Last Updated: <span class='date'>2017-12-20T11:55:57Z</span>
	    
	    <br>Latest Release: v3.11.0
	    
          </div>
        </div>
      </div>
      
      <div class="col-md-4">
        <div class="panel panel-default">
          <div class="panel-heading"><a href="http://www.sanger.ac.uk/science/tools/artemis"><h4>Artemis</h4></a></div>
	  <div class="panel-body">Artemis is a free genome viewer and annotation tool that allows visualization of sequence features and the results of analyses within the context of the sequence, and its six-frame translation</div>
	  <div class="panel-footer">
	    Last Updated: <span class='date'>2017-09-14T20:45:35Z</span>
	    
	    <br>Latest Release: v16.0.17
	    
          </div>
        </div>
      </div>
      
      <div class="col-md-4">
        <div class="panel panel-default">
          <div class="panel-heading"><a href="http://sanger-pathogens.github.io/circlator/"><h4>circlator</h4></a></div>
	  <div class="panel-body">A tool to circularize genome assemblies</div>
	  <div class="panel-footer">
	    Last Updated: <span class='date'>2017-12-12T15:11:05Z</span>
	    
	    <br>Latest Release: v1.5.3
	    
          </div>
        </div>
      </div>
      
    </div>
    <div class="row">
      <div class="col-md-12">
        <h2>Other Repos</h2>
      </div>
    </div>
    <div class="row">
      <div class="col-md-12">
        <table id="other_repos" class="table">
          <thead>
            <th>Repository</th>
            <th>Description</th>
	    <th>Score</th>
          </thead>
          <tbody>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/ariba">ariba</a></td>
              <td>Antimicrobial Resistance Identification By Assembly</td>
	      <td>47.37</td>
            </tr>
          
            <tr>
              <td><a href="http://sanger-pathogens.github.io/gubbins/">gubbins</a></td>
              <td>Rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences using Gubbins</td>
	      <td>42.952</td>
            </tr>
          
            <tr>
              <td><a href="http://sanger-pathogens.github.io/iva/">iva</a></td>
              <td>de novo virus assembler of Illumina paired reads</td>
	      <td>41.039</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Tradis">Bio-Tradis</a></td>
              <td>A set of tools to analyse the output from TraDIS analyses</td>
	      <td>3.919</td>
            </tr>
          
            <tr>
              <td><a href="http://dx.doi.org/10.1099/mgen.0.000056">snp-sites</a></td>
              <td>Finds SNP sites from a multi-FASTA alignment file</td>
	      <td>3.691</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/gff3toembl">gff3toembl</a></td>
              <td>Converts Prokka GFF3 files to EMBL files for uploading annotated assemblies to EBI</td>
	      <td>3.679</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Fastaq">Fastaq</a></td>
              <td>Python3 scripts to manipulate FASTA and FASTQ files</td>
	      <td>3.675</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/assembly-stats">assembly-stats</a></td>
              <td>Get assembly statistics from FASTA and FASTQ files</td>
	      <td>3.154</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/assembly_improvement">assembly_improvement</a></td>
              <td>Improve the quality of a denovo assembly by scaffolding and gap filling</td>
	      <td>2.998</td>
            </tr>
          
            <tr>
              <td><a href="http://companion.sanger.ac.uk">companion</a></td>
              <td>Pipeline for automatic eukaryotic parasite annotation</td>
	      <td>2.94</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/plasmidtron">plasmidtron</a></td>
              <td>Using kmers to find plasmids</td>
	      <td>2.921</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/pymummer">pymummer</a></td>
              <td>Python3 module for running MUMmer and reading the output</td>
	      <td>2.921</td>
            </tr>
          
            <tr>
              <td><a href="http://www.genedb.org/">GeneDB</a></td>
              <td>Web front-end for genomic data, developed at the Pathogen Genomics group, Wellcome Trust Sanger Institute</td>
	      <td>2.804</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/panito">panito</a></td>
              <td>Calculate genome wide average nucleotide identity (gwANI) for a multiFASTA alignment</td>
	      <td>2.718</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/ntd_website">ntd_website</a></td>
              <td>WTSI NTD website</td>
	      <td>2.682</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/mlst_check">mlst_check</a></td>
              <td>Multilocus sequence typing by blast using the schemes from PubMLST</td>
	      <td>2.606</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/saffrontree">saffrontree</a></td>
              <td>SaffronTree: Reference free rapid phylogenetic tree construction from raw read data</td>
	      <td>2.606</td>
            </tr>
          
            <tr>
              <td><a href="https://sanger-pathogens.github.io/phandango/">phandango</a></td>
              <td>an interactive viewer for populations of bacterial genomes linked by a phylogeny</td>
	      <td>2.352</td>
            </tr>
          
            <tr>
              <td><a href="http://sanger-pathogens.github.io/pathogens-vm/">pathogens-vm</a></td>
              <td>To build a virtual machine with some Sanger pathogens software installed</td>
	      <td>2.209</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/jobTree">jobTree</a></td>
              <td>Python based pipeline management software for clusters (but checkout toil: https://github.com/BD2KGenomics/toil, its successor)</td>
	      <td>2.025</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/progressiveCactus">progressiveCactus</a></td>
              <td>Distribution package for the Prgressive Cactus multiple genome aligner.  Dependencies are linked as submodules</td>
	      <td>2.011</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/update_pipeline">update_pipeline</a></td>
              <td>None</td>
	      <td>2.004</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/sanger-pathogens.github.io">sanger-pathogens.github.io</a></td>
              <td>Summary of Sanger Pathogen's Repos</td>
	      <td>1.976</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/MIDAS">MIDAS</a></td>
              <td>Front end for the HICF pathogen genome store</td>
	      <td>1.954</td>
            </tr>
          
            <tr>
              <td><a href="http://www.genedb.org/web-artemis/">Web-Artemis</a></td>
              <td>None</td>
	      <td>1.878</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/SnpEffWrapper">SnpEffWrapper</a></td>
              <td>Takes a VCF and applies annotations from a GFF using SnpEff</td>
	      <td>1.688</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/json2email">json2email</a></td>
              <td>Takes a jinja2 template and some json and sends an email</td>
	      <td>1.58</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/citation_reporter">citation_reporter</a></td>
              <td>Website to show your team's publications (which are in PubMed)</td>
	      <td>1.541</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/chado-export">chado-export</a></td>
              <td>Script for exporting organism sequences from a CHADO database to GFF file.</td>
	      <td>1.428</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/pathogen-informatics-training">pathogen-informatics-training</a></td>
              <td>None</td>
	      <td>1.416</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Metadata-Validator">Bio-Metadata-Validator</a></td>
              <td>Validate a genome metadata manifest against a checklist</td>
	      <td>1.214</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/seroba">seroba</a></td>
              <td>None</td>
	      <td>1.098</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/ariba-publication">ariba-publication</a></td>
              <td>Supplementary data and scripts for ARIBA publication</td>
	      <td>1.036</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Sequencescape-Schema">Bio-Sequencescape-Schema</a></td>
              <td>DBIC schema for the sequencescape warehouse database</td>
	      <td>1.006</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Track-Schema">Bio-Track-Schema</a></td>
              <td>DBIC schema for pathogens tracking databases</td>
	      <td>1.004</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-PacbioMethylation">Bio-PacbioMethylation</a></td>
              <td>Runs Pacbio methylation pipeline</td>
	      <td>1.003</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/companion-publication">companion-publication</a></td>
              <td>Accompanying information for the Companion publication</td>
	      <td>1.003</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/PySnpSites">PySnpSites</a></td>
              <td>Experiments optimising python (with Cython)</td>
	      <td>1.003</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/ABACAS2">ABACAS2</a></td>
              <td>ABACAS2, a tool for ordering and orientating biosequences along a reference</td>
	      <td>1.003</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/FMAP">FMAP</a></td>
              <td>Functional Mapping and Analysis Pipeline for metagenomics and metatranscriptomics studies</td>
	      <td>1.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/fastqcheck">fastqcheck</a></td>
              <td>Generate statistics on and validate fastq files</td>
	      <td>1.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/phandangoExampleData">phandangoExampleData</a></td>
              <td>The data underlying the examples on https://jameshadfield.github.io/phandango</td>
	      <td>1.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/structure">structure</a></td>
              <td>Copy of Structure 2.3.4 with patch applied to fix seg fault</td>
	      <td>1.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/vr-codebase">vr-codebase</a></td>
              <td>The overall codebase developed and used by the Vertebrate Resequencing group at the Sanger Institute</td>
	      <td>0.327</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Farmpy">Farmpy</a></td>
              <td> Python3 package to handle job submission to a compute farm</td>
	      <td>0.276</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/crawl2">crawl2</a></td>
              <td>A spiritual successor to Crawl</td>
	      <td>0.275</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/deployment">deployment</a></td>
              <td>None</td>
	      <td>0.238</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-RNASeq">Bio-RNASeq</a></td>
              <td>The new Sanger Pathogen Informatics RNA Seq analysis pipeline</td>
	      <td>0.221</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Deago">Bio-Deago</a></td>
              <td>None</td>
	      <td>0.219</td>
            </tr>
          
            <tr>
              <td><a href="http://companion.sanger.ac.uk">annot-web</a></td>
              <td>Web interface for Companion</td>
	      <td>0.2</td>
            </tr>
          
            <tr>
              <td><a href="http://ccb.jhu.edu/software/kraken/">kraken</a></td>
              <td>Kraken taxonomic sequence classification system</td>
	      <td>0.2</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/fastml">fastml</a></td>
              <td>Addtional functionality for fastml, see http://fastml.tau.ac.il</td>
	      <td>0.2</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-HICF-Schema">Bio-HICF-Schema</a></td>
              <td>None</td>
	      <td>0.171</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/pipelines_reporting">pipelines_reporting</a></td>
              <td>None</td>
	      <td>0.157</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/deago">deago</a></td>
              <td>None</td>
	      <td>0.147</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/setup_tracking">setup_tracking</a></td>
              <td>setup a vrtracking pipeline</td>
	      <td>0.137</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/reference_track">reference_track</a></td>
              <td>None</td>
	      <td>0.137</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/find_files_for_deletion">find_files_for_deletion</a></td>
              <td>None</td>
	      <td>0.137</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Farm_blast">Farm_blast</a></td>
              <td>Python3 module to run blast+ or blastall in parallel on an LSF compute farm</td>
	      <td>0.135</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/PathPipeConfTools">PathPipeConfTools</a></td>
              <td>Outputs details of all of the pathogen jobs in the pipeline</td>
	      <td>0.133</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Path-Find">Bio-Path-Find</a></td>
              <td>None</td>
	      <td>0.119</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/iva-publication">iva-publication</a></td>
              <td>Supplementary scripts and data for the IVA publication</td>
	      <td>0.1</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-AutomatedAnnotation">Bio-AutomatedAnnotation</a></td>
              <td>Perl module to take in an genomic assembly and produce annoation</td>
	      <td>0.1</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-ENA-DataSubmission">Bio-ENA-DataSubmission</a></td>
              <td>None</td>
	      <td>0.027</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-InterProScanWrapper">Bio-InterProScanWrapper</a></td>
              <td>None</td>
	      <td>0.023</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-ReferenceManager">Bio-ReferenceManager</a></td>
              <td>None</td>
	      <td>0.013</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/remove_blocks_from_aln">remove_blocks_from_aln</a></td>
              <td>None</td>
	      <td>0.012</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-VertRes-Config">Bio-VertRes-Config</a></td>
              <td>None</td>
	      <td>0.007</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-Metagenomics">Bio-Metagenomics</a></td>
              <td>None</td>
	      <td>0.003</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/madansi">madansi</a></td>
              <td>None</td>
	      <td>0.003</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/orderedPainting">orderedPainting</a></td>
              <td>None</td>
	      <td>0.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/Bio-VertRes-Permissions">Bio-VertRes-Permissions</a></td>
              <td>None</td>
	      <td>0.0</td>
            </tr>
          
            <tr>
              <td><a href="https://github.com/sanger-pathogens/HICF_checklist">HICF_checklist</a></td>
              <td>None</td>
	      <td>0.0</td>
            </tr>
          
          </tbody>
        </table>
      </div>
    </div>
    <div class="row">
      <div class="col-md-12">
        <p>Data collected: <span class='date'>2018-01-03T20:07:38Z</span></p>
      </div>
    </div>
    <script>
      $(document).ready(function() {
	function timeDeltaString (millis) {
          var seconds = millis / 1000;
	  if (seconds < 120) {
            return Math.round(seconds) + ' seconds ago';
	  }
	  var minutes = seconds / 60;
	  if (minutes < 120) {
            return Math.round(minutes) + ' minutes ago';
	  }
	  var hours = minutes / 60;
	  if (hours < 48) {
            return Math.round(hours) + ' hours ago';
	  }
	  var days = hours / 24;
	  if (days < 28) {
            return Math.round(days) + ' days ago';
	  }
	  var weeks = days / 7;
	  if (weeks < 12) {
            return Math.round(weeks) + ' weeks ago';
	  }
	  var months = weeks / 4;
	  return Math.round(months) + ' months ago';
        }
        var now = new Date();
        $( "span.date" ).each(function() {
          var date_string = this.textContent;
	  var date_obj = new Date(date_string);
          this.textContent = timeDeltaString(now - date_obj);
	});
	$('#other_repos').DataTable({
	  "order": [[ 2, "desc" ]]
	});
      });
    </script>
  </div>
</body>
